media/js/mozorg/contact-spaces.js

```diff
@@ -408,6 +409,7 @@
             var state = mozMap.getMapState();
             if (state === 'spaces') {
                 initContentId = $('#nav-spaces li.current').data('id');
+                mozMap.showSpace(state.url, initContentId);
```

Comment:
This is still causing a redundant ajax request on page load. `mozMap.showSpace()` does a lot more than setting the map looking at the code. I seem to remember addressing this ajax issue was how the regression snuck in to begin with.